gslib/commands/signurl.py

```diff
@@ -403,6 +404,39 @@ class UrlSignCommand(Command):
       subcommand_help_text={},
   )
 
+  _COMMAND_MAP = GcloudStorageMap(
+      gcloud_command=[
+          'alpha', 'storage', 'sign-url',
+          '--format=csv[separator="\\t"](resource, http_verb, expiration, signed_url)'
```

Comment:
What's this going to look like versus gsutil?

Reply:
The data itself is exactly the same. Here's gsutil's format: https://github.com/GoogleCloudPlatform/gsutil/blob/master/gslib/commands/signurl.py#L620

The only difference is the header row. Here's what gsutil puts there: https://github.com/GoogleCloudPlatform/gsutil/blob/master/gslib/commands/signurl.py#L558 

gcloud storage has tab separated `resource`, `http_verb`, `expiration`, `signed_url`. I didn't see a good way to print a row like this only once with the gcloud format flag, so to introduce a header row like gsutil's I think we'd need to do it with logic in the gcloud implementation. Think I should do that?

Reply:
Might be able to use a `multi` format where the first value is a `value` format with the desired header string and the second format is the CSV logic above